forge-game/src/main/java/forge/game/GameLogFormatter.java

```diff
@@ -247,9 +248,7 @@ public GameLogEntry visit(final GameEventAttackersDeclared ev) {
             if (sb.length() > 0) sb.append("\n");
             sb.append(localizer.getMessage("lblLogPlayerAssignedAttackerToAttackTarget", ev.player, Lang.joinHomogenous(attackers), k));
         }
-        if (sb.length() == 0) {
-            sb.append(localizer.getMessage("lblPlayerDidntAttackThisTurn").replace("%s", ev.player.toString()));
```

Comment:
Are you suggesting that someone not declaring attackers when they could have is not a useful log?